core/vm/contracts.libevm.go

```diff
@@ -223,9 +223,11 @@ func (args *evmCallArgs) env() *environment {
 	}
 
 	return &environment{
-		evm:      args.evm,
-		self:     contract,
-		callType: args.callType,
+		evm:       args.evm,
+		self:      contract,
+		callType:  args.callType,
+		rawCaller: args.caller.Address(),
+		rawSelf:   args.addr,
```

Comment:
I put a comment that these can be moved to caller site here https://github.com/ava-labs/libevm/pull/210#discussion_r2260142845, but this does not affect this PR.

Reply:
I'll look and respond there.